scripts/deploy-backend.sh

```diff
@@ -47,39 +47,22 @@ set_cf_envs()
   "DJANGO_SETTINGS_MODULE"
   "DJANGO_SU_NAME"
   "FRONTEND_BASE_URL"
-  "PROD_JWT_CERT"
-  "PROD_JWT_KEY"
   "LOGGING_LEVEL"
-  "PROD_ACR_VALUES"
-  "PROD_OIDC_OP_AUTHORIZATION_ENDPOINT"
-  "PROD_CLIENT_ASSERTION_TYPE"
-  "PROD_OIDC_RP_CLIENT_ID"
-  "PROD_OIDC_OP_ISSUER"
-  "PROD_OIDC_OP_JWKS_ENDPOINT"
-  "PROD_OIDC_OP_LOGOUT_ENDPOINT"
-  "PROD_OIDC_OP_TOKEN_ENDPOINT"
```

Comment:
I don't understand, what happens when deploying to prod then?

Reply:
Our CircleCI won't be deploying to prod. In the HHS CircleCI, these variables do not have the prefix and will deploy to prod as we had previously agreed on.